beniget/beniget.py

```diff
@@ -644,9 +734,15 @@ def visit_Assign(self, node):
     def visit_AnnAssign(self, node):
         if node.value:
             dvalue = self.visit(node.value)
-        dannotation = self.visit(node.annotation)
+        if not self.future_annotations:
+            dannotation = self.visit(node.annotation)
+        else:
+            self._defered_annotations[-1].append(
```

Comment:
See previous comment, looks like unless they are at top-level, these assign should be dropped (?)